packages/worker/src/utilities/email.ts

```diff
@@ -162,6 +164,47 @@ export async function sendEmail(
       contents: opts?.contents,
     }),
   }
+  if (opts?.attachments) {
+    const baseUrl = appContext.getPlatformURL()
+    const attachments = await Promise.all(
+      opts.attachments?.map(async attachment => {
+        const isFullyFormedUrl =
```

Comment:
nit: typically we'd refer to URLs with a scheme and a domain as "absolute" and paths as "relative" but it's not really a problem, don't feel the need to change anything.